packages/secure_storage/amplify_secure_storage/pigeons/messages.macos.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// To regenerate, run `make pigeons`.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(
```

Comment:
I considered just copying the output in the Makefile command, but there is a minor difference between the two files since `TaskQueue` isn't supported for MacOS.